cmd/component-codegen/cmd/generator/rbac.go

```diff
@@ -0,0 +1,39 @@
+package generator
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+func addKubeBuilderRBAC(logger *logrus.Logger, componentName string) error {
+	lc := strings.ToLower(componentName)
+	comments := []string{
+		componentName,
+		fmt.Sprintf("+kubebuilder:rbac:groups=components.platform.opendatahub.io,resources=%ss,verbs=get;list;watch;create;update;patch;delete", lc),
+		fmt.Sprintf("+kubebuilder:rbac:groups=components.platform.opendatahub.io,resources=%ss/status,verbs=get;update;patch", lc),
+		fmt.Sprintf("+kubebuilder:rbac:groups=components.platform.opendatahub.io,resources=%ss/finalizers,verbs=update", lc),
+		"Please delete this line after adding more RBAC markers ...",
```

Comment:
```suggestion
		"TODO: Delete this line after adding more RBAC markers ...",
```

^or maybe just leave that line out...will there always be more RBAC markers to add manually?

Reply:
Done 👍